### ISSUE TITLE
jaydata 1.2 Merge on SharePoint 

### ISSUE THREAD

**Issue by RainerAtSpirit** [jaystack/jaydata#53]
I'm following along your Merge example at http://jaydata.org/blog/restful-singleton-operations-with-jaydata-1.2.

``` javascript
MyApp.context.Tasks.first(null, null, function(item){ 
    MyApp.context.attach(item); 
    item.TaskName = "Updated via JayData", 
    MyApp.context.saveChanges();
    })

```

but the sytem throws the following error.

```
{
"error": {
"code": "", "message": {
"lang": "en-US", "value": "Error processing request stream. Error encountered in converting the value from request payload for property 'Owshiddenversion' to type 'Int32', which is the property's expected type. See inner exception for more detail."
}
}
}
```

In addition to the expected Id and TaskName I see an additional `Owshiddenversion` entry in the payload.

```
{"Id":1,"TaskName":"Updated via JayData","Owshiddenversion":""}
```

The service file was generated by JaySvcUtil and wasn't manually modified. 

``` javascript
  'Owshiddenversion': { type: 'Edm.Int32', nullable: true, concurrencyMode: $data.ConcurrencyMode.Fixed },       
```

If I remove `,"Owshiddenversion":""}` from the payload and replay via fiddler the Merge operation produces the expected 204 result.

See full details at https://gist.github.com/3749453

**Comment by PeterAronZentai** [comment]
Uh oh.. This is bad news :( good that you found it. And it's fix is a blatant one, already running the unit tests. 
We are about to rerelease v1.2 this fix tomorrow afternoon. Thank you so much for bringing this.
Apparently WCF and LightSwitch is more tolerant but this is clearly an error.

**Comment by PeterAronZentai** [comment]
This has been resolved with 1.2.1
